lib/utils.pm

```diff
@@ -1349,6 +1349,33 @@ sub _handle_login_not_found {
     die "unknown error, system couldn't boot. Detailed bootup log:\n$error_details";
 }
 
+=head2 _handle_firewall
+
+ _handle_firewall();
+
+Internal helper function used by C<reconnect_mgmt_console>.
+
+=cut
+sub _handle_firewall {
+    select_console 'root-console';
+    if (script_run("iptables -S | grep 'A input_ext.*tcp.*dport 59.*-j ACCEPT'", 30) != 0) {
+        wait_quit_zypper;
+        my $ret;
```

Comment:
Is it a bug on s390x?
why locked there if no active daemon?

Reply:
not bug, this is caused by firewall upgrade